backend/baseclass.pm

```diff
@@ -748,7 +748,7 @@ sub reload_needles (@) {
     # called from testapi::set_var, so read the vars
     bmwqemu::load_vars();
 
-    $_->unregister() for needle->all();
+    $_->unregister() for needle::all();
```

Comment:
This looks quite strange to me, considering all you did was add `()`. But it seems to work I guess

Reply:
The syntax `needle->all()` kinda worked but was never intended as `needle::all` was a free function, not an object method